test/unit/specs/connectors/lcdClientMock.spec.js

```diff
@@ -102,6 +102,23 @@ describe(`LCD Client Mock`, () => {
     expect(res.length).toBe(3)
   })
 
+  it(`persists a sent tx to myself`, async () => {
```

Comment:
We don't extend the lcdclientmock anymore

Reply:
the coverage of lcdClientMock decreased once I've changed some test to avoid running send from here, once Ive done that I had to cover that dead piece of code to have the green ✅ from CCI

Reply:
how about instead ignoring coverage on lcdClientMock?

Reply:
I think we should keep it working and tested until we don't drop it

Reply:
But one good thing would be stopping importing it from the spec files touched, so that PR will be easier, I'll drop that import from transactions.spec.js